### DISCUSSION TITLE
Include details of the current page in the pagination object

### DISCUSSION THREAD

**Discussion by edwardhorsford** [11ty/eleventy#4070] [Enhancement Queue]
Recently lots was added to the `pagination` object for next and previous page. 

But there isn't the same parity for the **current** page.

So I'd like to request `pagination.page` and pagination.href` have entries added for `current` as well as next and previous.

**Comment by zachleat** [comment]
> This repository is now using [lodash style issue management](https://twitter.com/samselikoff/status/991395669016436736) for enhancements. This means enhancement issues will now be closed instead of leaving them open.

> View [the enhancement backlog here](https://github.com/11ty/eleventy/issues?utf8=%E2%9C%93&q=label%3Aneeds-votes+sort%3Areactions-%2B1-desc+). Don’t forget to upvote the top comment with 👍!
